tsconfig.compile.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Is the idea here to produce different sets of compiled sources for es2018 and es2015 environments?  Just trying to keep up with what's happening :)

Reply:
This is for running tests in node only.

Reply:
If possible, let's just re-target the whole library to es2015